src/components/main-dashboard/weather-details/Humidity.jsx

```diff
@@ -1,9 +1,46 @@
 import React from 'react';
+import ProgressBar from '@ramonak/react-progress-bar';
+import { useWeather } from '../../WeatherProvider';
+import humidityIcon from '../../../images/humidityIcon.png';
 
 export default function Humidity() {
+  const { currentWeather } = useWeather();
+  if (!currentWeather) return null;
+  let humidityDescription = 'Ideal humidity levels';
+  let humidityBarColor = '#28a745'; // default to green color
```

Comment:
I need to declare it let because it changes depending on what the humidity levels are. Are you referring to using the `const` variable declaration or something else?

Reply:
I was referring to defining them as something like, 

`const green = '#28a745'`
`const red = '#ff0000'`
etc.  

and then referencing them when needed, like line 10
`let humidityBarColor = red`;

Reply:
Oh okay I see! Yeah that is more readable for sure, I'll change that!